csFastFloat/Utils/Utils.cs

```diff
@@ -121,6 +121,12 @@ internal static bool is_space(byte c)
         return Unsafe.As<byte, bool>(ref b);
     }
 
+    [MethodImpl(MethodImplOptions.AggressiveInlining)]
+    internal static bool is_ansi_space(char c)
+    {
+        return c <= byte.MaxValue && is_space(unchecked((byte)c));
```

Comment:
```suggestion
        return c <= byte.MaxValue && is_space((byte)c);
```
`unchecked` is by default, so this could be omitted.